### ISSUE TITLE
Change examples to use Selector.with(payload) instead of Command::new()

### ISSUE THREAD

**Issue by arthmis** [linebender/druid#1735]
It seems it wouldn't be idiomatic? druid to use Command::new() directly and users should be pushed to use `Selector` to create commands. Currently the examples use Command new, so it would be nice to change any uses of `Command::new()` to `MySelector.with()`. Is this right? @cmyr 

old way:
```rust
    const ExampleSelector: Selector<u64> = Selector::new("example-selector");
    Command::new(ExampleSelector, 45u64, Target::Auto);
```
now it should be
```rust
    const ExampleSelector: Selector<u64> = Selector::new("example-selector");
    ExampleSelector.with(45u64);
```

**Comment by cmyr** [comment]
Yes, I think this is the most ergonomic way to use these API. Thanks!

**Comment by arthmis** [comment]
Closed with #1761
